BeeKit/Managers/GoalManager.swift

```diff
@@ -95,33 +138,24 @@ public actor GoalManager {
             return
```

Comment:
When `updateGoalsFromJson` is called with the list of strings (goal names), this function exits early in this guard.

Where are the calls
`GET /users/u/goals/g.json`
for each of the goals corresponding to the goal names listed in the goalResponse when fetched via the user endpoint?